### PR TITLE
Bugfix/ZEB-894 empty platform icons

### PR DESCRIPTION
**Pull request by MitskevichVS**

### CODE REVIEW

File: client/app/containers/tests-runs/tests-runs.controller.js [hunk 1]
```diff
@@ -551,6 +557,7 @@ const testsRunsController = function testsRunsController(
         return vm.zafiraWebsocket.subscribe('/topic/' + authService.tenant + '.testRuns', function (data) {
             const event = getEventFromMessage(data.body);
             const testRun = angular.copy(event.testRun);
+            [testRun.platformIcon, testRun.platformVersion] = testsRunsService.refactorPlatformData(testRun.config);
```
Comment (danymill) [inline client/app/containers/tests-runs/tests-runs.controller.js]: Please di it after vars definition

File: client/app/components/blocks/test-run-card/test-run-card.directive.js [hunk 2]
```diff
@@ -442,9 +442,7 @@
                 }
 
                 function normalizePlatformData() {
-                    if (vm.testRun.config) {
-                        [vm.testRun.platformIcon, vm.testRun.platformVersion] = testsRunsService.refactorPlatformData(vm.testRun.config);
-                    }
+                    [vm.testRun.platformIcon, vm.testRun.platformVersion] = testsRunsService.refactorPlatformData(vm.testRun.config);
                 }
```
Comment (danymill) [inline client/app/components/blocks/test-run-card/test-run-card.directive.js]: This action should be removed from this directive like you have done it in `testsSessionCardController`

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/containers/tests-runs/tests-runs.controller.js (hunk 1 above)
Comment (MitskevichVS) [reply client/app/containers/tests-runs/tests-runs.controller.js]: fixed

File: client/app/components/blocks/test-run-card/test-run-card.directive.js (hunk 2 above)
Comment (MitskevichVS) [reply client/app/components/blocks/test-run-card/test-run-card.directive.js]: fixed

**Review by danymill — APPROVED** [review]

**Pull request merged**